src/components/Navbar.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+import style from '../style/NavBar.module.css';
 
 const Nav = () => (
-  <header>
+  <header className={style.nav}>
     <h1>Bookstore CMC</h1>
-    <nav>
-      <Link to="/">Books</Link>
-      <Link to="/categories">Categories</Link>
+    <nav className={style.menue}>
+      <Link className={style.link} to="/">Books</Link>
+      <Link className={style.link} to="/categories">Categories</Link>
     </nav>
+    <button type="button" className={style.navIconBtn}>
+      <FaUser />
+    </button>
   </header>
```

Comment:
- [ ] Kindly ensure that your Header sticks as much as possible to the template design.
### Your design
![Screenshot from 2023-03-03 15-04-26](https://user-images.githubusercontent.com/83042742/222729895-f38d258b-e680-4ba0-ad98-464d3c9dd513.png)
### Template design
![Screenshot from 2023-03-03 15-14-48](https://user-images.githubusercontent.com/83042742/222730336-f6ab9ffe-d2ed-4595-86c0-c17490ba5d78.png)